settings.yml

```
@@ -0,0 +1,126 @@
+  first_date: "2020-01-01"
+  # Last date to simulate
+  last_date: "2025-10-01"
+
+flags:
+  # Running on CS cluster
+  CS: false
+  # NAG availability
+  nag: false
+  # Knitro availability
+  knitro: false
+
+  # Test Derivatives of ED
+  TestED: false
+  # Test Objective Function Derivatives
+  TestObj: true
+  # Test Derivatives of Constraints
+  TestEstimationConstraint: true
+  # Test Programs to Compute OmegaBar
+  TestOmega: false
+  # Test Computation of Equilibrium
+  TestComputeEquilibrium: false
+  # Allow Default (tue = sigmaBar > 0 and allow default; false = sigmaBar = 0 for all industries)
+  allowDefault: true
+
+  # Use data from mat files
+  data_from_mat: false
+
+  # Random initial values for parameters
+  random_initial_values: false
+
+elasticities:
+  consumption:
+    - [3, 2.0]
+  EU_export_demand:
+    - [3, 2.0, 2.0]
+  ROW_export_demand:
+    - [3, 2.0, 2.0]
+  production:
+    - [0.4, 0.75, 2.0]
+  investment:
+    - [0.4, 2.0]
+
+files:
+  input_dir: input
+  output_dir: output
+
+  input_derived_dir: derived
+
+  inputs:
+    base:
+      InputOutput: "uk_input_output2019.xlsx"
+      Imports: "uk_input_output2019.xlsx"
+      mergeCodes64: "mergeCodes64.xlsx"
+      mergeCodes105: "mergeCodes105.csv"
+
+    derived:
+      # Time series: income of two types (lo, hi) in two scenarios (0,1) = (noCovid, covid)
+      loIncome0: "noCollegeIncome0.csv"
+      hiIncome0: "collegeIncome0.csv"
+      loIncome1: "noCollegeIncome1.csv"
+      hiIncome1: "collegeIncome1.csv"
+
+      # Time series: hours worked of two types (lo, hi) in two scenarios (0,1)
+      loHours0: "noCollegeHours0.csv"
+      hiHours0: "collegeHours0.csv"
+      loHours1: "noCollegeHours1.csv"
+      hiHours1: "collegeHours1.csv"
+
+      # Time series: capital in two scenarios (0,1)
+      capital0: "capital0.csv"
+      capital1: "capital1.csv"
+
+      # Time series: inventories (one scenario only)
+      inventory: "derived/{version}/inventories.csv"
+
+      # Base year: depreciation rates
+      depreciation: "depreciation.csv"
+
+      # Financial data
+      RiskFreeRate: "interestRate.csv"
+      assets: "firmData.csv"
+      DefaultSigma: "modelResults.csv"
+
+      # Forecast scenarios
+      gdp: "gdp_forecast.csv"
+      turnoverS0: "turnover_s0.csv"
+      turnoverS1: "turnover_s1.csv"
+  outputs:
+    BaseData: "ukdata.mat"
+    parms: "parms.mat"
+    logp: "logp.mat"
+    LogY: "logy.mat"
+    NoCovidMAT: "forecastNoCovid.mat"
+    NoCovidCSV: "forecastNoCovid.csv"
+    CovidCSV: "cge_results.csv"
+    CovidMAT: "turnover_model.mat"
+    TradeCSV: "net_exports.csv"
+    SimulateEconomy3MAT: "simEcon3.mat"
+    SimulateEconomy3CSV: "simEcon3.csv"
+    CompStatsResults: "compStats.mat"
+
+initial_params:
+  # Scaling factors for the parameters
+  hLOScale: 1
+  hHIScale: 1
+  kScale: 1
+  yScale: 1
+
+  P1Tilde: 1
+  PX1Tilde: 1
+  P2Tilde: 1
+  PX2Tilde: 1
+
+  q0: 1.0
+  kOmega: 10
+  lambda: 0.12 # see Bernanke et al. page 1368
```

I'm up for discussing this but I'd also say we use descriptive variable names. As an example, something like `household_labor` rather than `h`. While it makes the math paper to code translation more cumbersome, it makes the code to human language translation much easier, and I think that is going to be a lot more valuable. I'm hoping it will also force us to understand the code we are writing a bit better.

```suggestion
  labor_low_scale: 1
  labor_high_scale: 1
  capital_scale: 1
  firm_output_scale: 1

  eu_price_index: 1
  eu_export_price_index: 1
  row_price_index: 1
  row_export_price_index: 1

  beginning_price_of_capital: 1.0
  leverage_grid_resolution: 10
  assets_destroyed_in_bankrupcy: 0.12 # see Bernanke et al. page 1368
```